proto/expr.proto

```diff
@@ -102,6 +102,7 @@ message ExprNode {
     BIT_LENGTH = 230;
     OVERLAY = 231;
     REGEXP_MATCH = 232;
+    REGEXP_REPLACE = 320;
```

Comment:
Also, is this number identifier `320` in `proto/expr.proto` fits for `regexp_replace`?
I could change this to be compatible if this is not fit.

Reply:
Maybe `279`? to continue the number within the group.

Reply:
Ahh good idea🥰, I directly used the one after `FORMAT` without seeing `278`🤣